### PR TITLE
fix delete clusterFile

### PR DESCRIPTION
**Pull request by bxy4543**
fix cobra cmd;fix delete clusterFile

### CODE REVIEW

File: sealer/cmd/login.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ var loginCmd = &cobra.Command{
 	Use:   "login",
 	Short: "login image repositories",
 	Long:  `sealer login registry.cn-qingdao.aliyuncs.com -u [username] -p [password]`,
+	Args:  cobra.MinimumNArgs(1),
```
Comment (allencloud) [inline sealer/cmd/login.go:38]: It seems that the number of args here should be exactly 1.

**Comment by allencloud** [comment]
Thanks for your contribution. In terms of args number, you could refer to https://github.com/spf13/cobra/blob/eb3b6397b1b5d1b0a2cd66a9afe0520f480c0a87/args.go#L60-L87 for guidance. @bxy4543

File: apply/cloud_applier.go [hunk 2]
```diff
@@ -101,10 +101,9 @@ func (c *CloudApplier) Delete() error {
 	}
 	if err := utils.RemoveFileContent(common.EtcHosts, fmt.Sprintf("%s %s", host, common.APIServerDomain)); err != nil {
 		logger.Warn(err)
-		return nil
 	}
 
-	if err := utils.CleanFiles(common.DefaultKubeconfigDir, common.GetClusterWorkDir(c.ClusterDesired.Name), common.KubectlPath); err != nil {
+	if err := utils.CleanFiles(common.DefaultKubeconfigDir, common.GetClusterWorkDir(c.ClusterDesired.Name), common.TmpClusterfile, common.KubectlPath); err != nil {
```
[the remark this answers is not in the document]
Comment (bxy4543) [reply apply/cloud_applier.go:106]: exec cloud apply,sealer will fetch kubectl to local

File: sealer/cmd/login.go [hunk 3]
```diff
@@ -52,4 +53,6 @@ func init() {
 	rootCmd.AddCommand(loginCmd)
 	loginCmd.Flags().StringVarP(&loginConfig.RegistryUsername, "username", "u", "", "user name for login registry")
 	loginCmd.Flags().StringVarP(&loginConfig.RegistryPasswd, "passwd", "p", "", "password for login registry")
+	_ = loginCmd.MarkFlagRequired("username")
```
[the remark this answers is not in the document]
Comment (fanux) [reply sealer/cmd/login.go:56]: @bxy4543 if return error, should not ignore it.

**Review by fanux — APPROVED** [review]

**Pull request merged**